user-auth-api.py

```
@@ -0,0 +1,108 @@
+    'database': 'users'
+}
+
+@app.route('/login', methods=['POST'])
+def login():
+    username = request.form.get('username')
+    password = request.form.get('password')
+    
+    # SQL Injection vulnerability
+    conn = mysql.connector.connect(**DB_CONFIG)
+    cursor = conn.cursor()
+    query = f"SELECT * FROM users WHERE username='{username}' AND password='{password}'"
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)